### ISSUE TITLE
Can't use a comma within the tagger name

### ISSUE THREAD

**Issue by bennyb** [modxcms/Tagger#144]
It seems you can't use a comma in the tagger name, if for example you create a tag called
Tagger: 2.0.0-pl
MODX: 3.0.5

one, two & three.

Can can see this fine within the tagger select dropdown while editing a resource, but if you assign it then everything after the comma is ignored.  Using the above example it would try and assign a tag called one.  If you save and then refresh nothing is actually saved.

There is the label field which could work if this can then be used within the resource tagger dropdown and selected tags as you have to rename the name field for the tag which can mean it doesn't make sense to the user without the comma.  

Also if that field can't have a comma than this should get stripped out when creating the tag to avoid users adding this in by mistake.

**Comment by matdave** [comment]
This has been the case since inception. However you can add a "Label" to the tag with a comma

**Comment by bennyb** [comment]
> This has been the case since inception. However you can add a "Label" to the tag with a comma 

Yeah I see the label, issue with that is there's no way of displaying that to a user who's doing the tagging. So without a comma the tag name can make no sense. Obviously displays ok when viewed on Frontend but be good to be able to specify to use label on the tagger tab on the resource. 

Also be good if tagger could strip the comma from the name as no warning so users who can add tags themselves will get confused as to why their tags can't be used on a resource as they appear as normal.

**Comment by matdave** [comment]
You can intentionally add commas to create multiple tags. This is why we don't allow commas in tag names.
